### ISSUE TITLE
`space-before-keywords` false positive

### ISSUE THREAD

**Issue by npasquet** [eslint/eslint#4449]
With ``` eslint@1.9.0```

and rules:
```javascript
{
  'rules': {
    'curly': [2, 'multi-line', 'consistent'],
    'space-before-keywords': [2, 'never']
  }
}
``` 

I'm getting the error:

```javascript
error    Unexpected space before keyword "else"                 space-before-keywords
```

when writing my conditional block like that: 

```javascript
if(currentPath === 'delivered') query.delivered = 'true';
else query.running = 'all';
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.
